application/static/landregistry/assets/stylesheets/landregistry.scss

```diff
@@ -161,4 +161,18 @@ dl {
   dd {
     float: left;
   }
-}
\ No newline at end of file
+}
+
+.deed-document--bounds {
```

Comment:
As much as I like BEM (I do!), none of the surrounding code is written using BEM conventions so I wonder if these should perhaps be `.deed-document-bounds` and `..deed-document-divider`?